### ISSUE TITLE
Cannot make this project

### ISSUE THREAD

**Issue by autodidaddict** [flopezluis/rabbitmq-throttling-exchange#1]
The make file simply includes a file called `umbrella.mk`, which does not exist in the repository.

**Comment by alphashuro** [comment]
@autodidaddict did you manage to make the project?

**Comment by autodidaddict** [comment]
No, I was not able to build it. It's clearly missing the umbrella.mk file.

**Comment by autodidaddict** [comment]
It's a shame, because this would've solved a real problem I've got.

**Comment by alphashuro** [comment]
i'm also facing a real problem and i've started copying over stuff from the rabbitmq example plugin https://github.com/rabbitmq/rabbitmq-metronome 

what problem are you having if i might ask? i'm trying to implement throttling based on the routing key, and i've made the changes required to this plugin but now i need to just test it and see if it works

**Comment by autodidaddict** [comment]
Mine is pretty simple.. I've got a downstream system that can't handle more than a certain request rate. Rather than building something terrible like a local counter, I want a distributed rate throttler, so if messages only arrive on my queue at a fixed rate, I won't exceed that for my downstream system.

**Comment by alphashuro** [comment]
ahh that seems like it can already be solved by the plugin in its current state, however i would encourage you to try to get it running as well, if i find a solution i will let you know
